packages/storybook/stories/va-date.stories.jsx

```diff
@@ -40,70 +40,85 @@ const Template = ({ label, name, required, error, value }) => {
 
 const CustomValidationTemplate = ({ label, name, required, error, value }) => {
   const [dateVal, setDateVal] = useState(value);
-  const [currentYear, currentMonth, currentDay] = (dateVal || '').split('-');
-
-  if (currentYear < 1900 || currentYear > 2122) {
-    error = 'Please enter a year between 1900 and 2122';
-  }
+  const [errorVal, setErrorVal] = useState(error);
   const today = new Date();
   // new Date as YYYY-MM-DD is giving the day prior to the day select
   // new Date as YYYY MM DD is giving the correct day selected
   const dateInput = new Date(dateVal.split('-').join(' '));
-  if (dateInput <= today) {
-    error = 'Date must be in the future';
+  function handleDateBlur() {
+    if (dateInput <= today) {
+      setErrorVal('Date must be in the future');
+    } else {
+      setErrorVal('');
+    }
   }
+
   return (
     <>
       <VaDate
         label={label}
         name={name}
         required={required}
-        error={error}
+        error={errorVal}
         value={dateVal}
-        onDateBlur={e => console.log(e, 'DATE BLUR')}
+        onDateBlur={() => handleDateBlur()}
         onDateChange={e => setDateVal(e.target.value)}
       />
+      <hr />
       <div>
         This example has some custom validation logic built out to detect
-        changes made to the select and input fields. If the criteria below is
-        not met an error message will show:
+        changes made to the input fields that fire when the component is blurred
+        ie: focus is removed from the component. If the criteria below is not
+        met an error message will show:
         <ul>
           <li>Cannot have blank values</li>
-          <li>The Year falls outside of the range of 1900-2122</li>
+          <li>Month and Day are not valid</li>
+          <li>The Year falls outside of the range of 1900-2200</li>
```

Comment:
Should this be updated to current year + 100? Using a year of 2199 is showing a "Please enter a valid date" error.

Reply:
Good catch yes the date utility it year + 100 need to update the verbiage to match that

Reply:
Updated thanks!